crates/re_sdk/src/recording_stream.rs

```diff
@@ -1095,10 +1095,11 @@ impl RecordingStream {
     #[cfg(feature = "data_loaders")]
     pub fn log_file_from_path(
         &self,
-        settings: &re_data_source::DataLoaderSettings,
         filepath: impl AsRef<std::path::Path>,
+        entity_path_prefix: Option<EntityPath>,
```

Comment:
why not first put the entity path like with all other logging functions 🤔 

edit: right, because that messes up languages with default arguments